inc/sainexthopgroup.h

```diff
@@ -299,6 +299,17 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_LABEL,
 
+    /**
+     * @brief Weighted multi path configuration mode.
+     * false: Nexthop group is programmed with repeated member entries proportional to their weight
```

Comment:
When the SAI implementation repeats next hop group member entries, should the SAI_SWITCH_ATTR_AVAILABLE_NEXT_HOP_GROUP_MEMBER_ENTRY attribute be updated to reflect the count of the repeated entries, or should it be based on the number of unique members in the ASIC_DB?